# HG changeset patch
# User andrew
# Date 1378914912 -3600
#      Wed Sep 11 16:55:12 2013 +0100
# Node ID 56f4e181b8d485a01f125f2c681159cc940f6c26
# Parent  09f439dfd7b9d6e022f7af71c328b5b4b6a36c23
Set ZERO_BUILD in flags.make so it is set on rebuilds

diff --git a/make/bsd/makefiles/buildtree.make b/make/bsd/makefiles/buildtree.make
--- openjdk/hotspot/make/bsd/makefiles/buildtree.make
+++ openjdk/hotspot/make/bsd/makefiles/buildtree.make
@@ -215,6 +215,7 @@
 	echo "HOTSPOT_BUILD_USER = $(HOTSPOT_BUILD_USER)"; \
 	echo "HOTSPOT_VM_DISTRO = $(HOTSPOT_VM_DISTRO)"; \
 	echo "OPENJDK = $(OPENJDK)"; \
+	echo "ZERO_BUILD = $(ZERO_BUILD)"; \
 	echo; \
 	echo "# Used for platform dispatching"; \
 	echo "TARGET_DEFINES  = -DTARGET_OS_FAMILY_\$$(Platform_os_family)"; \
diff --git a/make/linux/makefiles/buildtree.make b/make/linux/makefiles/buildtree.make
--- openjdk/hotspot/make/linux/makefiles/buildtree.make
+++ openjdk/hotspot/make/linux/makefiles/buildtree.make
@@ -195,7 +195,6 @@
 	$(BUILDTREE_COMMENT); \
 	echo; \
 	echo "Platform_file = $(PLATFORM_FILE)" | sed 's|$(GAMMADIR)|$$(GAMMADIR)|'; \
-	echo "JVM_VARIANTS = $(JVM_VARIANTS)"; \
 	sed -n '/=/s/^ */Platform_/p' < $(PLATFORM_FILE); \
 	echo; \
 	echo "GAMMADIR = $(GAMMADIR)"; \
@@ -210,6 +209,7 @@
 	echo "HOTSPOT_BUILD_USER = $(HOTSPOT_BUILD_USER)"; \
 	echo "HOTSPOT_VM_DISTRO = $(HOTSPOT_VM_DISTRO)"; \
 	echo "OPENJDK = $(OPENJDK)"; \
+	echo "ZERO_BUILD = $(ZERO_BUILD)"; \
 	echo; \
 	echo "# Used for platform dispatching"; \
 	echo "TARGET_DEFINES  = -DTARGET_OS_FAMILY_\$$(Platform_os_family)"; \
